notification/slack/blocks.go

```diff
@@ -0,0 +1,96 @@
+package slack
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/slack-go/slack"
+	"github.com/target/goalert/alert"
+)
+
+func alertIDAndStatusSection(id int, status string) *slack.HeaderBlock {
+	var s string
+	if status == "triggered" {
+		s = "Unacknowledged"
+	} else if status == "active" {
+		s = "Acknowledged"
+	} else {
+		s = "Closed"
+	}
```

Comment:
We do have a closed status to check for as well, and in this case we're hoping that everything is 1-to-1. What do you think of a switch, and on default panicking? We should also probably use the `AlertStatus` type instead of `string` as the func parameter

Reply:
https://github.com/target/goalert/pull/1358/commits/1f80a4ef6cd2a51ac7ee8cd4e27e4c5053c162ab#diff-f5ee0dc56091cb36e70eb6e95a5bc4906ef20cd69cf14f7de917018578c1332cR13